easybuild/easyconfigs/g/gffread/gffread-0.10.6-GCCcore-7.3.0.eb

```
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'gffread'
+version = '0.10.6'
+
+homepage = 'https://github.com/gpertea/%(name)s'
+description = """GFF/GTF parsing utility providing format conversions,
+region filtering, FASTA sequence extraction and more."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['http://ccb.jhu.edu/software/stringtie/dl/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['760b7c61542dc65caec0e7551e51fa43c0b2e0da536d8e2f37a472ac285a04e1']
+
+builddependencies = [('binutils', '2.30')]
+
+buildopts = " release"
+
+files_to_copy = ['%(namelower)s', 'LICENSE']
+
+modextrapaths = {'PATH': ''}
+
+sanity_check_paths = {
+    'files': ['%(namelower)s'],
```

The name is already in lower case:

```suggestion
    'files': ['%(name)s'],
```